server/seed.js

```diff
@@ -17,10 +17,33 @@ async function createUser(){
 	})
 };
 
+async function createListing(){
+	await Listing.destroy({where: {} });
+	await Listing.create({
+		skill: 'plumber',
+		isActive: true,
+		serviceDetails: 'bathroom sink is clogged',
+		suggestedPrice: 80.00,
+		location: 'Brooklyn, NY',
+	})
+};
```

Comment:
This is totally fine here, but it's a good example of separating the concerns of multiple tickets. You had an outstanding ticket for adding the listing sample data and made these changes on a separate branch, but then you've also made them here. 

Since both this and the listing branch were branched from a previous version of Master, they both will be merged back into master, but one of them will be merged first. So the changes above will be compared to the changes you made in the other branch. If they aren't exactly the same, even with formatting, there will be a merge conflict. 

We can talk more about that on our call tomorrow if you want, but in the meantime this is completely fine.